bin/pycbc_live

```diff
@@ -369,13 +369,17 @@ class LiveEventManager(object):
         """
         from ligo.gracedb.rest import GraceDb
         logging.info('Creating GraceDB session')
-        # Set up dict for args that reload the certificate if it will expire in the reload_buffer time
-        # These args are documented here: https://ligo-gracedb.readthedocs.io/en/latest/api.html#ligo.gracedb.rest.GraceDb
-        # Because we do not change any of the request session values when running the code, it should remain thread safe. 
+        # Set up dict for args that reload the certificate if it will expire in the
+        # reload_buffer time. These args are documented here:
+        # https://ligo-gracedb.readthedocs.io/en/latest/api.html#ligo.gracedb.rest.GraceDb
+        # Because we do not change any of the request session values when running the
+        # code, it should remain thread safe. 
         gdbargs = {'reload_certificate': True, 'reload_buffer': 300}
-        self.gracedb = GraceDb(args.gracedb_server, **gdbargs) if args.gracedb_server else GraceDb(**gdbargs)
```

Comment:
I had proposed the following to Xan instead:
```python
gdbargs = {'reload_certificate': True, 'reload_buffer': 300}
if args.gracedb_server:
    gdbargs['service_url'] = args.gracedb_server
self.gracedb = GraceDb(**gdbargs)
```

Reply:
Yes, that seems a bit more straightforward, particularly as the API declares `service_url` as a kwarg rather than a positional arg (`service_url='https://gracedb.ligo.org/api/'`)